graylog2-web-interface/webpack.config.js

```diff
@@ -171,6 +171,5 @@ if (TARGET === 'test') {
 }
 
 if (Object.keys(module.exports).length === 0) {
-  console.error('Running in default mode');
```

Comment:
Would it make sense to log this on [`console.trace()`](https://developer.mozilla.org/en-US/docs/Web/API/Console/trace)?